packages/services/src/sports-nfl-ingester.service.ts

```diff
@@ -429,7 +500,10 @@ export class NflIngesterService {
       score.GlobalGameID,
       tx,
     );
-    const wasAlreadyFinal = existingGame?.status === "final";
+    const wasAlreadyFinal =
```

Comment:
here's where we make sure that games with a missing `endTime` or `winner` are properly scored.